src/Pkg.jl

```diff
@@ -87,7 +87,7 @@ a package, also inside that package.
 # Examples
 ```julia
 Pkg.add("Example") # Add a package from registry
-Pkg.add(PackageSpec(name="Example", version="0.3")) # Specify version
+Pkg.add(PackageSpec(name="Example", version="0.3.1")) # Specify version
```

Comment:
This change is not really necessary

Reply:
I thought `0.3.1` might be more familiar and helpful to new users, as Julia packages have `x.y.z` version numbering.
Also for someone unfamiliar with versioning and `Pkg` specifics it might be confusing.
For example, I'm unsure whether `version="0.3"` means a specific version because the versioning system only has `x.y` numbering, or whether it means add the _latest_ `0.3` version, say `0.3.7` rather than earlier versions.

Reply:
You make a good point. I'm in favor of this.

Reply:
Maybe just add a second line and have two example:
```
Pkg.add(PackageSpec(name="Example", version="0.3")) # Specify version; latest release in the 0.3 series
Pkg.add(PackageSpec(name="Example", version="0.3.1")) # Specify version; exact release
```